frontend/.eslintrc.js

```diff
@@ -36,8 +36,10 @@ module.exports = {
   },
   plugins: ["@typescript-eslint", "react", "sonarjs", "@blueprintjs"],
   rules: {
+    "@typescript-eslint/camelcase": 0,
```

Comment:
I think I'm turning off the camelCase lint rule, since we just have so many of them from API responses 😆